legend-engine-xts-json/legend-engine-xt-json-runtime/pom.xml

```diff
@@ -110,6 +110,10 @@
             <groupId>org.finos.legend.engine</groupId>
             <artifactId>legend-engine-shared-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.finos.legend.engine</groupId>
+            <artifactId>legend-engine-xt-relationalStore-executionPlan</artifactId>
+        </dependency>
```

Comment:
Per my comment on the other PR, we need to avoid bringing specific stores to "core" functionality like externalize.  The goal is that externalize works with "any" store, and we need to find a way to dispatch to respective store modules, if we need to.